media-autobuild_suite.bat

```diff
@@ -2072,9 +2084,9 @@ echo.---------------------------------------------------------------------------
 echo.install %1 bit compiler
 echo.-------------------------------------------------------------------------------
 if %CC%==clang (
-    if "%1"=="32" (
-        set prefix=mingw-w64-clang-i686-
-    ) else set prefix=mingw-w64-clang-x86_64-
+    if "%1"=="64" (
+        set prefix=mingw-w64-clang-x86_64-
+    )
```

Comment:
```suggestion
    if "%1"=="64" set prefix=mingw-w64-clang-x86_64-
```

Tbh, I'm pretty sure this if check can be deleted since we check that `CC==clang && build32==yes` is an invalid state, so `call :getmingw 32` should never be called when `CC==clang`.